.vimrc

```diff
@@ -439,23 +439,23 @@
       " Rails
       " {{{
       nnoremap <C-p> :completefunc()<CR>
-      nnoremap <F6> :Rails 
```

Comment:
I think, that `:Rails, :Rake, Rgenerate` stayed the same. At least, I didn't find them in [vim-rails docs](https://github.com/tpope/vim-rails/blob/master/doc/rails.txt). Could you change them back?